packages/remix/package.json

```diff
@@ -12,8 +12,7 @@
   "scripts": {
     "build": "node build.js",
     "test-integration-once": "pnpm test test/test.js",
-    "test": "jest --env node --verbose --bail --runInBand",
-    "test-unit": "pnpm test test/build.test.ts"
```

Comment:
The unit tests were deleted since they test basically the same as the integration tests. However, `await import()` breaks Jest currently, so they needed to be removed.